### PR TITLE
nfs-ganesha: fix scm section, add absolute paths

### PR DESCRIPTION
**Pull request by alimaredia**
Made changes to scm section to configure both git
repos.

Made all references to "dist" dir absolute paths

Made sure directories being git cleaned are
actual git repos

Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

File: nfs-ganesha/build/build_rpm [hunk 1]
```diff
@@ -41,12 +41,12 @@ cmake -DCMAKE_BUILDER_TYPE=Maintainer -DUSE_FSAL_RGW=YES -DUSE_FSAL_CEPH=YES $WO
 ## Create the source rpm
 echo "Building SRPM"
 rpmbuild \
-    --define "_sourcedir ./dist" \
+    --define "_sourcedir $WORKSPACE/dist" \
     --define "_specdir ." \
```
Comment (alimaredia) [inline nfs-ganesha/build/build_rpm]: @b-ranto, do i need to change the paths for _specdir, _builddir, _scrpmdir, and _rpmdir to $WORKSPACE/dist in this command?

File: nfs-ganesha/config/definitions/nfs-ganesha.yml [hunk 2]
```diff
@@ -61,13 +61,9 @@ If this is checked, then the binaries will be built and pushed to chacra even if
           type: label-expression
           name: AVAILABLE_DIST
           values:
-            - centos6
```
Comment (alimaredia) [inline nfs-ganesha/config/definitions/nfs-ganesha.yml]: @alfredodeza were these values necessary? I'm confused about which distros get triggered to build when I push the "Build" button in "Build With Parameters" section of the jenkins UI.

File: nfs-ganesha/config/definitions/nfs-ganesha.yml [hunk 3]
```diff
@@ -83,26 +104,8 @@ If this is checked, then the binaries will be built and pushed to chacra even if
...
-          url: https://github.com/nfs-ganesha/nfs-ganesha-debian.git
-          # Use the SSH key attached to the ceph-jenkins GitHub account.
-          credentials-id: '39fa150b-b2a1-416e-b334-29a9a2c0b32d'
-          branches:
-            - xenial-nfs-ganesha-ceph-fsals
-          skip-tag: true
-          wipe-workspace: true
-          basedir: "nfs-ganesha-debian"
+      - nfs-ganesha
+      - nfs-ganesha-debian
```
Comment (alfredodeza) [inline nfs-ganesha/config/definitions/nfs-ganesha.yml]: this looks correct

File: nfs-ganesha/config/definitions/nfs-ganesha.yml (hunk 2 above)
Comment (alfredodeza) [inline nfs-ganesha/config/definitions/nfs-ganesha.yml]: this section would list everything that you think the project is able to build. That is why this is named `AVAILABLE_DIST`. It allows for filtering the list from user input, e.g. 'just build in centos7'. You can remove centos6 if that will never ever be built (I think that may be the case here)

File: nfs-ganesha/build/build_rpm (hunk 1 above)
Comment (b-ranto) [inline nfs-ganesha/build/build_rpm]: @alimaredia It depends on where your .spec file is, where you want your (src.)rpms to be located, etc. The . is used in place of $PWD in these cases if you were asking about that.

**Pull request merged**